### PR TITLE
Updating golang to version 1.14.8

### PR DESCRIPTION
**Pull request by Toasterson**
Small update with only some files changed.

### CODE REVIEW

**Comment by Toasterson** [comment]
The diffs are huge, as COMPONENT_VERSION is not output anymore by oi-userland scripts as variable.

File: components/developer/golang-114/Makefile [hunk 1]
```diff
@@ -30,15 +30,15 @@ PATCH_EACH_ARCHIVE= 1
...
 COMPONENT_SUMMARY=		The Go Programming Language
 COMPONENT_SRC=			$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_PROJECT_URL=		https://golang.org/
 COMPONENT_ARCHIVE=		go$(COMPONENT_VERSION).src.tar.gz
 COMPONENT_ARCHIVE_URL=		https://storage.googleapis.com/golang/$(COMPONENT_ARCHIVE)
-COMPONENT_ARCHIVE_HASH=		sha256:6d643e46ad565058c7a39dac01144172ef9bd476521f42148be59249e4b74389
+COMPONENT_ARCHIVE_HASH=		sha256:d9a613fb55f508cf84e753456a7c6a113c8265839d5b7fe060da335c93d6e36a
 COMPONENT_LICENSE=		BSD-style, Patent Grant
 COMPONENT_LICENSE_FILE=		$(COMPONENT_NAME).license
 MEDIATEDUSRSHAREMAN1DIR=	$(PROTO_DIR)/usr/share/golang/$(COMPONENT_MAJOR_VERSION)/man/man1
```
Comment (AndWac) [inline components/developer/golang-114/Makefile:44]: When I build this on my machine, I'll get some differences in the sample-manifest:
╰─➤  diff sample-manifest.p5m sample-manifest.p5m.new
202a203
> file path=usr/lib/golang/1.14/gocache/0f/0fe420e661120639374e665c7fbdafaa4366cd233667c1a3a9a02b51a7d02577-d
204,205c205,206
< file path=usr/lib/golang/1.14/gocache/54/544bc14607d45da0faab42175a487f82354441d587db68258c8a3a53148ba2bc-a
< file path=usr/lib/golang/1.14/gocache/8f/8fd1e6f79fd26d922c3a68022f03ffef975e099adc74b68599c9466aeb843048-d
---
> file path=usr/lib/golang/1.14/gocache/38/387075b5e194b3e0f13d0321d90cfbc050ae253f5e6de8696793b4e4a3d0e59f-a
> file path=usr/lib/golang/1.14/gocache/82/82231e244123a775dddf0a3d29b3d4b5e7c2378c786bae5f7fc8ab4451db6951-a
207,209c208
< file path=usr/lib/golang/1.14/gocache/b1/b14b0972a75a4a246791cedc44c7b085ac028912720386d121f2234b1038db00-a
< file path=usr/lib/golang/1.14/gocache/bd/bdd65f74951c0e46d2ed06636512028c7feb3a0a3a6e67e08265c31c629ab225-a
< file path=usr/lib/golang/1.14/gocache/cc/cc23f724798b791a3c873ff4c5e2e243941133c656ea99ce32cf18b1cbbe9746-a
---
> file path=usr/lib/golang/1.14/gocache/df/dfca21abc2a6dedba179dfa70715c1d82c1351a3ad2cf1065272f6428cd295ec-a
210a210
> file path=usr/lib/golang/1.14/gocache/e9/e9bc43cb5ff6693f03146a721e0a60ccfaa4a27f83a63cfc2ec00e62ed69d6e6-a

These differences let "gmake publish" fail.
I expect that something similar will happen on our jenkins.

Comment (Toasterson) [reply components/developer/golang-114/Makefile:44]: Fixed with the newest commit

**Comment by AndWac** [comment]
Now it publishes. Please squash the commits before integration!

**Pull request merged**